tests/test_replica.py

```diff
@@ -31,7 +31,7 @@
 from rucio.common.schema import get_schema_value
 from rucio.common.utils import clean_surls, generate_uuid, parse_response
 from rucio.core.config import set as cconfig_set
-from rucio.core.did import add_did, attach_dids, get_did, get_did_atime, list_files, set_status
+from rucio.core.did import add_did, attach_dids, get_did, get_did_atime, get_did_access_cnt, list_files, set_status
```

Comment:
Please sort the imports to pass the linter check (you can do this automatically via Ruff, see https://rucio.github.io/documentation/contributing#1-prerequisite)